common/src/main/java/com/sedmelluq/discord/lavaplayer/natives/NativeLibLoader.java

```diff
@@ -70,12 +127,20 @@ public static void load(Class<?> resourceBase, String name, String systemTypeFil
    * @param name Name of the library
    * @throws LinkageError When loading the library fails
    */
-  public static void load(Class<?> resourceBase, String name) {
+  public static void load(Class<?> resourceBase, String name) throws LinkageError, UnsatisfiedLinkError {
     synchronized (loadedLibraries) {
       if (!loadedLibraries.contains(name)) {
-        try {
-          System.load(extractLibrary(resourceBase, name).toFile().getAbsolutePath());
+        String libPath = System.getProperty("lavaplayer." + name + ".path");
```

Comment:
Absolute path expected, so makes sense to still wrap `new File(...path...).getAbsolutePath()` around it in case people pass relative paths for whatever reason.

Reply:
I tested relative paths in the latest commit, works fine after adding what's on line 78 below (as of c0ae2dccaf637ed490fd769f6d69d3398da57b01)